mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -273,17 +273,21 @@ def runConfigWithMLIR(config: ConvConfiguration):
         outs, errs = p3.communicate()
 
 def runConfigWithMIOpenDriver(commandLine, envs):
-    # remove the result file generated by rocprof in previous benchmarking
-    os.system("rm "+BENCHMARKING_RESULT_FILE_NAME)
-    profilerCommand = [ROCPROF, '--stats', MIOPEN_DRIVER, *commandLine, '-V', '0']
+    MIOpenDriverCommand = [MIOPEN_DRIVER, *commandLine, '-V', '0']
     print("Running MIOpen Benchmark: ", ' '.join(commandLine))
-    # invoke rocprof + MIOpenDriver.
-    p1 = subprocess.Popen(profilerCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=envs)
+    # invoke MIOpenDriver.
+    p1 = subprocess.Popen(MIOpenDriverCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=envs)
     # get output.
     try:
         outs, errs = p1.communicate(timeout=180)
         if len(errs) > 0:
-            print("MIOpen benchmark produced errors: ", errs.decode('utf-8'))
```

Comment:
It's still useful to have the errors printed out?

Reply:
Previously the error message was consumed by rocprof silently and it was reflected in the final report (as FAILED or --). Do we really want to print these error messages when running the tests? Maybe people will check the report anyway to see if there are unsupported configs.

Reply:
It may be good for debugging to know if there's a failure failure vs an unsupported failure vs something else.

But we could put that behind some flavor of `--debug` switch, or drop it entirely. Not too important

Reply:
Let's keep it. It's useful that this can help differentiate between different type of failures right from the log.